nixos/tests/containers-names.nix

```
@@ -0,0 +1,37 @@
+import ./make-test-python.nix ({ pkgs, ...} : {
+  name = "containers-names";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ patryk27 ];
+  };
```

```suggestion
import ./make-test-python.nix ({ pkgs, lib,  ...} : {
  name = "containers-names";
  meta = {
    maintainers = with lib.maintainers; [ patryk27 ];
  };
```

Don't use pkgs.lib, but `lib` directly.

---

Fix'd, thanks 🙁